### PR TITLE
add createNode

### PR DESCRIPTION
**Pull request by uxitten**
#632

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -265,13 +265,13 @@ The virtual DOM allows us to write code as if the entire document is redrawn on
 Hyperapp provides <samp>hyperapp.h</samp> to create virtual nodes. The <samp>h</samp> function takes an element's name or a function that returns a virtual node (see [Components](#components)), optional attributes and optional array of children elements.
```
Comment (jorgebucaran) [inline README.md]: <samp>hyperapp.createNode</samp>

**Comment by jorgebucaran** [comment]
@uxitten Could you please alias createNode to `h` only in the test files? Also please rebase with master. 🙏

File: README.md (hunk 1 above)
Comment (uxitten) [reply README.md]: fixed

**Comment by uxitten** [comment]
i fixed again document and run npm format command 

Ok?

**Comment by uxitten** [comment]
Need rebase?

**Comment by jorgebucaran** [comment]
https://github.com/hyperapp/hyperapp/pull/633#issuecomment-370019159


> @uxitten Could you alias createNode to h only in the test files?

File: README.md [hunk 2]
```diff
@@ -43,14 +43,14 @@ This example assumes you are using a JavaScript compiler like [Babel](https://ba
 }
 ```
 
-JSX is a language syntax extension that lets you write HTML tags interspersed with JavaScript. Because browsers don't understand JSX, we use a compiler to transform it into <samp>hyperapp.h</samp> function calls, our virtual DOM builder function.
+JSX is a language syntax extension that lets you write HTML tags interspersed with JavaScript. Because browsers don't understand JSX, we use a compiler to transform it into <samp>hyperapp.createNode</samp> function calls, our virtual DOM builder function.
 
 ```jsx
 const view = (state, actions) =>
```
Comment (jorgebucaran) [inline README.md]: Let's add:

```js
import { createNode as h } from "hyperapp"
```

File: README.md [hunk 3]
```diff
@@ -43,14 +43,14 @@ This example assumes you are using a JavaScript compiler like [Babel](https://ba
...
-  h("div", {}, [
-    h("h1", {}, state.count),
-    h("button", { onclick: () => actions.down(1) }, "-"),
-    h("button", { onclick: () => actions.up(1) }, "+")
+  createNode("div", {}, [
+    createNode("h1", {}, state.count),
+    createNode("button", { onclick: () => actions.down(1) }, "-"),
+    createNode("button", { onclick: () => actions.up(1) }, "+")
   ])
```
Comment (jorgebucaran) [inline README.md]: Use `h` here.

File: README.md [hunk 4]
```diff
@@ -187,10 +187,10 @@ This operation doesn't replace the entire DOM tree, but only update the parts of
 
 ```js
 const view = (state, actions) =>
```
Comment (jorgebucaran) [inline README.md]: Use JSX here.

**Comment by uxitten** [comment]
ok

File: README.md (hunk 2 above)
Comment (frenzzy) [reply README.md]: why?

**Comment by uxitten** [comment]
what should I do now?

**Comment by jorgebucaran** [comment]
@uxitten Maybe rebase again, sorry! I changed the docs a bit today. 😓

**Comment by uxitten** [comment]
what is problem in this PR?

**Comment by jorgebucaran** [comment]
@uxitten There is no problem. I just still haven't made up my mind about changing h→createNode. We need to collect more feedback on #632.

**Comment by uxitten** [comment]
ok very good

**Comment by okwolf** [comment]
Since #632 was closed should this be closed also?

**Comment by jorgebucaran** [comment]
Sorry, @uxitten, but this didn't make it to the cut in the end! 👋

**Pull request closed without merging**